src/tlitest/tlitest-run

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
Do not constrain yourself to `/bin/sh` unnecessarily, as it's not always Bash there (especially on Debian/Ubuntu). Just write `/bin/bash` always, unless you really intend to limit yourself to POSIX shell features only.

Reply:
Good point about other distros.  I wasn't thinking about that.  I'l switch all the scripts to specify bash.